### ISSUE TITLE
MakeLeftHanded and FlipWindingOrder post-processing steps are interfering with aiAnimMesh

### ISSUE THREAD

**Issue by rickomax** [assimp/assimp#2267]
I'm implementing morph (blend-shapes) into the FBX parser and I've seen that if I pass these two post-processing steps (MakeLeftHanded and FlipWindingOrder) to Assimp, it won't update the aiAnimMesh structures, resulting in distorted results. I'll try to fix this one myself.

**Comment by rickomax** [comment]
@kimkulling I've fixed that in TriLib. There are a couple of things I did there that we can merge into the main repository. I'll contact you when I have it ready to merge.

**Comment by fire** [comment]
Was this fixed? I'm getting blocked by this.

**Comment by rickomax** [comment]
Yes, it was fixed on #2293
